<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>681</width>
    <height>492</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>收信规则</string>
  </property>
  <property name="windowIcon">
   <iconset resource="mailmgr.qrc">
    <normaloff>:/icon/hello.jpg</normaloff>:/icon/hello.jpg</iconset>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>651</width>
     <height>471</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab_ok">
    <attribute name="title">
     <string>黑名单设置</string>
    </attribute>
    <widget class="QTableWidget" name="blackList">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>271</width>
       <height>391</height>
      </rect>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::DefaultContextMenu</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QHeaderView::section{background:rgb(85, 170, 255)};</string>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="textElideMode">
      <enum>Qt::ElideMiddle</enum>
     </property>
     <column>
      <property name="text">
       <string>邮件地址</string>
      </property>
     </column>
    </widget>
    <widget class="QLineEdit" name="mailno">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>80</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="addBtn1">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>130</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>添加</string>
     </property>
    </widget>
    <widget class="QPushButton" name="saveBtn1">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>130</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>保存</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkLocal">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>350</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>开启本地过滤规则</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>270</y>
       <width>321</width>
       <height>151</height>
      </rect>
     </property>
     <property name="title">
      <string>过滤规则</string>
     </property>
     <widget class="QPushButton" name="okBtn1">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>70</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkDb">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>120</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>开启数据库过滤规则</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBlack">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>开启黑名单过滤</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>50</y>
       <width>321</width>
       <height>151</height>
      </rect>
     </property>
     <property name="title">
      <string>手动添加</string>
     </property>
     <widget class="QComboBox" name="domain_1">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>30</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string/>
       </property>
      </item>
      <item>
       <property name="text">
        <string>@163.com</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>@gmail.com</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>@hotmail.com</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>@qq.com</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>@126.com</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>@yeah.net</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QPushButton" name="delBtn1">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>210</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>删除</string>
     </property>
    </widget>
    <zorder>groupBox</zorder>
    <zorder>groupBox_2</zorder>
    <zorder>blackList</zorder>
    <zorder>mailno</zorder>
    <zorder>addBtn1</zorder>
    <zorder>saveBtn1</zorder>
    <zorder>checkLocal</zorder>
    <zorder>delBtn1</zorder>
   </widget>
   <widget class="QWidget" name="tab_no">
    <attribute name="title">
     <string>白名单设置</string>
    </attribute>
    <widget class="QTableWidget" name="whiteList">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>271</width>
       <height>391</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QHeaderView::section{background:rgb(85, 170, 255)};
</string>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="textElideMode">
      <enum>Qt::ElideMiddle</enum>
     </property>
     <column>
      <property name="text">
       <string>邮件地址</string>
      </property>
     </column>
    </widget>
    <widget class="QLineEdit" name="mailok">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>80</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="domain_2">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>80</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>@163.com</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>@gmail.com</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>@hotmail.com</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>@qq.com</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>@126.com</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>@yeah.net</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="addBtn2">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>130</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>添加</string>
     </property>
    </widget>
    <widget class="QPushButton" name="saveBtn2">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>130</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>保存</string>
     </property>
    </widget>
    <widget class="QPushButton" name="delBtn2">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>210</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>删除</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>50</y>
       <width>291</width>
       <height>151</height>
      </rect>
     </property>
     <property name="title">
      <string>手动添加</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>270</y>
       <width>321</width>
       <height>151</height>
      </rect>
     </property>
     <property name="title">
      <string>过滤规则</string>
     </property>
     <widget class="QPushButton" name="okBtn2">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>60</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkWhite">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>开启白名单</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkLocal_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>80</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>开启本地过滤规则</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <zorder>groupBox_3</zorder>
    <zorder>whiteList</zorder>
    <zorder>mailok</zorder>
    <zorder>domain_2</zorder>
    <zorder>addBtn2</zorder>
    <zorder>saveBtn2</zorder>
    <zorder>delBtn2</zorder>
    <zorder>groupBox_4</zorder>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="mailmgr.qrc"/>
 </resources>
 <connections/>
</ui>
